src/main/java/org/jenkinsci/plugins/docker/commons/fingerprint/DockerFromFingerprintFacet.java

```diff
@@ -9,7 +9,7 @@
  * @author Kohsuke Kawaguchi
  */
 public class DockerFromFingerprintFacet extends DockerRunPtrFingerprintFacet {
-    public DockerFromFingerprintFacet(Fingerprint fingerprint, long timestamp) {
-        super(fingerprint, timestamp);
+    DockerFromFingerprintFacet(Fingerprint fingerprint, long timestamp, String imageId) {
```

Comment:
Making package-private under the assumption that `DockerFingerprints` covers the legitimate use cases. If not, this could be restored to `public`.